package.json

```diff
@@ -118,7 +118,7 @@
         "lexical.server.releasePathOverride": {
           "scope": "window",
           "type": "string",
-          "markdownDescription": "The path to the lexical release the extension should start. The path should point to a folder containing the `start_lexical.sh` script. Note that setting this to any value will disable automatic installation of Lexical."
+          "markdownDescription": "The path to the lexical executable. The path should point to a folder containing the `start_lexical.sh` or an executable file. Note that setting this to any value will disable automatic installation of Lexical."
```

Comment:
Thinking more about this, I think we should actually deprecate pointing to a folder. I don't think there's much of a point in supporting both behaviors; I'd rather the option be as clear and simple as can be.

If you agree, a simple doc change here and in the README will be fine for now. Maybe I'll add an alert later to ask users with a folder path configured to update to a file path.

Reply:
Ok, I did the backwards compatibility thing so as to not break people's editors. I would favor the approach of popping up an alert if they point to a folder.

Reply:
@Blond11516 can we validate that the setting is correct and display an error in the settings UI if it's not?

Reply:
I'll have to double check, but I think the most we can get from the settings UI is basic type validation (ints, strings, enums, etc.).

What I had in mind is a notification in the bottom right like this
![8jpzzvhhc8091-3977497654](https://github.com/lexical-lsp/vscode-lexical/assets/44734484/6d05d6ff-cb98-4d4b-bfe1-c95b0c728101)

These allow custom actions, so we could also provide an option to not show the warning again (or not since we'd want users to migrate their config asap).

I'll merge this as is either way, deprecation can come later.

Reply:
Issue to discuss and track this: https://github.com/lexical-lsp/vscode-lexical/issues/38